ip/ip.sh

```diff
@@ -0,0 +1,3 @@
+curl ipinfo.io/json -q | jq
```

Comment:
Fails with errors:
`jq: command not found`
`curl: (23) Failed writing body`

Reply:
You need to install jq for that by 
```bash
sudo apt-get install jq
```

Reply:
Add that to the script. Other users might not have it installed, just like I don't.
Your script should work out of the box.

Reply:
How is it now ?